spec/hosts/roles/hexrot_spec.rb

```diff
@@ -32,16 +32,85 @@
           it { is_expected.to contain_class('docker') }
 
           %w[
+            anaconda
+            mate
             profile::core::common
             profile::core::debugutils
             profile::core::docker
             profile::core::docker::prune
             profile::core::ni_packages
+            profile::core::x2go
```

Comment:
This duplicates `include_examples 'x2go packages', os_facts: os_facts` above.

Reply:
removed 👍🏻